src/ray/protobuf/reporter.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package ray.rpc;
+
+import "src/ray/protobuf/common.proto";
+
+message GetProfilingStatsRequest {
+  // PID of the worker process.
+  uint32 pid = 1;
+  // Duration of the profiling in seconds.
+  int32 duration = 2;
+}
+
+message GetProfilingStatsReply {
+  // Profiling stats.
+  string profiling_stats = 1;
+  string stdout = 2;
+  string stderr = 3;
+}
+
+// Service for inter-node-manager communication.
```

Comment:
This should be `Service for communicating with the reporter.py process on a remote node`